src/frontend/src/components/RecordSearch/Field.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+interface Props {
+  name: string;
+  label: string;
+  content: string;
+  coda: string;
+  onChange: Function; // arg: value
+  errorMessage: string;
+  required: boolean;
+}
+
+interface State {
+  hasInput: boolean;
+}
+
+export default class Field extends React.Component<Props, State> {
+  state: State = {
+    hasInput: false
+  }
+
+  handleInputChange = (e: React.BaseSyntheticEvent) => {
+    let fieldContent : string = e.target.value
+    this.setState(
+      {
+        hasInput: fieldContent.length > 0
+      }
+      )
+
+    this.props.onChange(e.target.value);
+  };
+
+  render() {
+    return(
+      <div className="w-100 w-third-ns w-25-l mb3">
+        <label htmlFor={this.props.name} className="db mb1 fw6">
+          {this.props.label} <span className= "fw2 f6">{this.props.coda}</span>
+        </label>
+        <input
+          value={this.props.content}
+          id={this.props.name}
+          type="text"
+          className="w-100 b--black-20 br2 br-0-ns br--left-ns pa3"
```

Comment:
Each of the inputs have unique classes to give them the appearance of being connected:

First Name: `w-100 b--black-20 br2 br-0-ns br--left-ns pa3`
Middle Name: `w-100 br2 br0-ns b--black-20 pa3`
Last Name: `w-100 b--black-20 br2 bl-0-ns br--right-ns pa3`
DOB wrapper div: `w-100 w-third-ns w-25-l pl2-l mb3`
DOB: `w-100 pa3 br2 b--black-20`

![Screen_Shot_2020-02-14_at_4_43_03_PM-2](https://user-images.githubusercontent.com/6887982/74574446-570e8e00-4f49-11ea-835f-8b891a6d4c0d.png)


Otherwise looking great!

Reply:
ah, gotcha!